pkg/generate/code/set_resource.go

```diff
@@ -1123,6 +1186,7 @@ func setResourceForContainer(
 			targetFieldName,
 			targetVarName,
 			targetShapeRef,
+			targetSetCfg,
 			sourceVarName,
 			sourceShapeRef,
 			indentLevel,
```

Comment:
Oh wow these are getting out of hand :/

Reply:
Yeah at first glance looks like we might be able to refactor by passing `Field` type in directly instead of each of its fields or maybe extend `Field` type with these methods